lib/Eclair/EIR/Codegen.hs

```diff
@@ -1,5 +1,5 @@
 module Eclair.EIR.Codegen
-  ( CodegenT(..)
+  ( CodegenT()
```

Comment:
Type synonyms usually are exported like below: 

```suggestion
  ( CodegenT
```

(Apply this suggestion to fix it.)